include/libsemigroups/paths.hpp

```diff
@@ -736,55 +379,57 @@ namespace libsemigroups {
   //! Constant
   template <typename Node1, typename Node2>
   [[nodiscard]] paths::algorithm
-  number_of_paths_algorithm(WordGraph<Node1> const& d, Node2 source) noexcept {
-    (void) d;
+  number_of_paths_algorithm(WordGraph<Node1> const& wg, Node2 source) noexcept {
+    (void) wg;
     (void) source;
     return paths::algorithm::acyclic;
   }
 
   //! \ingroup paths_group
   //! Returns the number of paths from a source node.
   //!
+  //! \param wg the WordGraph
   //! \param source the source node.
   //!
   //! \returns A value of type `uint64_t`.
   //!
   //! \throws LibsemigroupsException if \p source is not a node in the
-  //! digraph.
+  //! word graph.
   //!
   //! \complexity
   //! At worst \f$O(nm)\f$ where \f$n\f$ is the number of nodes and \f$m\f$
-  //! is the out-degree of the digraph.
+  //! is the out-degree of the word graph.
   //!
   //! \warning If the number of paths exceeds 2 ^ 64, then return value of
   //! this function will not be correct.
   template <typename Node1, typename Node2>
-  [[nodiscard]] uint64_t number_of_paths(WordGraph<Node1> const& d,
+  [[nodiscard]] uint64_t number_of_paths(WordGraph<Node1> const& wg,
                                          Node2                   source);
 
   //! \ingroup paths_group
-  //! Returns the \ref algorithm used by number_of_paths().
+  //! Returns the paths::algorithm used by number_of_paths().
   //!
   //! Returns the algorithm used by number_of_paths() to compute the number
   //! of paths originating at the given source node with length in the range
   //! \f$[min, max)\f$.
   //!
+  //! \param wg the WordGraph
```

Comment:
```suggestion
  //! \param wg the WordGraph.
```
(and the same for the params below)